packages/docs/site/docs/developers/05-local-development/04-wp-playground-cli.md

```diff
@@ -213,21 +265,56 @@ The Playground CLI can also be controlled programmatically from your JavaScript/
 Using the `runCLI` function, you can specify options like the PHP and WordPress versions. In the example below, we request PHP 8.3, the latest version of WordPress, and to be automatically logged in. All supported arguments are defined in the `RunCLIArgs` type.
 
 ```TypeScript
-import { runCLI, RunCLIArgs, RunCLIServer } from "@wp-playground/cli";
+import { runCLI, RunCLIArgs} from "@wp-playground/cli";
 
-let cliServer: RunCLIServer;
+const cliServer = await runCLI({
+  command: 'server',
+  php: '8.3',
+  wp: 'latest',
+  login: true,
+} as RunCLIArgs);
+```
 
-cliServer = await runCLI({
+To execute the code above, you can set your preferred method. A simple way to execute this code is to save it as a `.ts` file and run it with a tool like `tsx`. For example: `tsx my-script.ts` if you have the package installed on your project or `npx tsx my-script.ts` to use it once without installing the package.
```

Comment:
Let's simplify it:

`````suggestion
Run the code above using your preferred TypeScript runtime, e.g. `tsx`:

```sh
npx tsx my-script.ts
```

`````

Also, if we remove `as RunCLIArgs` it should just work in Node.js and we wouldn't have to explain the TypeScript runtime.